README.md

```diff
@@ -5,6 +5,9 @@ Each document in an index is a single action (create, update, or comment) taken
 that could conceivably be part of the given time range, and decomposing that issue into a series of actions. Those actions are then written
 to a series of .tsv (tab-separated value) files that are uploaded to an Imhotep shardbuilder.
 
+We have included some apachejira sample queries [here](http://opensource.indeedeng.io/imhotep/docs/sample-data/). You can gain insights
+from the data returned from your query, which can empower you to make data-driven decisions.
```

Comment:
Instead of the generic statement, can you say something more specific about the kinds of insights those samples enable?